test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
```

Comment:
Are these comments in reference to my changes to `dev-resources/logback.xml`?

In general I agree with you that changing the global logging config for one test is not a good idea.  However, I've come to think that the correct / best practice default config for test logging is to have things set to `warn`.  That way any test where you are explicitly expecting a warning, you can take steps to comply with the Wolfe Principle, but any other test that is unexpectedly logging a warning will show up in the test output rather than being silenced.  This seems like a pretty valuable way to catch unexpected changes, e.g. from upstream libraries, and defaulting the test logs to `error` seems like a bad pattern to me.

So I would actually advocate for that change to `logback.xml` even in the absence of this PR.

Let me know if you disagree with that, or if I've misunderstood these comments entirely :)